Screens/RegistrationScreen.js

```diff
@@ -94,116 +93,119 @@ class RegistrationScreen extends LocalizeComponent {
-              />
-              {this.props.registration.BirthDateError ? (
-                <TextError style={styles.error}>{this.t(this.props.registration.BirthDateError)}</TextError>
-              ) : null}
+                <TextInput
+                  textContentType="telephoneNumber"
+                  placeholder={this.t("TELEPHONE")}
+                  placeholderTextColor="#707070"
+                  style={styles.input}
+                  onChangeText={(txt) => this.props.enterRegistrationTelephone(txt)}
+                  onBlur={this.props.validateRegistrationTelephone}
+                />
+                {this.props.registration.TelephoneError ? (
+                  <TextError style={styles.error}>{this.t(this.props.registration.TelephoneError)}</TextError>
+                ) : null}
+                  <DatePicker
+                    style={styles.input}
+                    androidMode='spinner'
+                    defaultDate={new Date(2018, 4, 4)}
+                    locale={this.props.languageCode}
+                    timeZoneOffsetInMinutes={120}
+                    modalTransparent={false}
+                    animationType={"fade"}
+                    androidMode={"default"}
+                    placeHolderText={this.t("SELECT DATE OF BIRTH")}
+                    textStyle={{ color: "green" }}
+                    placeHolderTextStyle={{
+                      color: "#707070", fontWeight: "bold"
+                    }}
+                    onDateChange={(txt) => { this.props.enterRegistrationBirthdate(txt); }}
+                    disabled={false}
+                  />
+                  {this.props.registration.BirthDateError ? (
+                    <TextError style={styles.error}>{this.t(this.props.registration.BirthDateError)}</TextError>
+                  ) : null}
 
+                <TextInput
+                  textContentType="emailAddress"
+                  placeholder={this.t("EMAIL")}
+                  placeholderTextColor="#707070"
+                  style={styles.input}
+                  onChangeText={(txt) => this.props.enterRegistrationEmail(txt)}
+                  onBlur={this.props.validateRegistrationEmail}
+                />
+                {this.props.registration.EmailError ? (
+                  <TextError style={styles.error}>{this.t(this.props.registration.EmailError)}</TextError>
+                ) : null}
 
-              <TextInput
-                textContentType="emailAddress"
-                placeholder={this.t("EMAIL")}
-                placeholderTextColor="#707070"
-                style={styles.input}
-                onChangeText={(txt) => this.props.enterRegistrationEmail(txt)}
-                onBlur={this.props.validateRegistrationEmail}
-              />
-              {this.props.registration.EmailError ? (
-                <TextError style={styles.error}>{this.t(this.props.registration.EmailError)}</TextError>
-              ) : null}
+                <TextInput
```

Comment:
as i remember, we already have customized component to text inputs. find and use it

Reply:
I can't user our custom text input, because custom text input doesn't have all needed props

Reply:
what you think about component modification to make it reusable in both places?